.gitignore

```diff
@@ -14,6 +14,9 @@
 .classpath
 .project
 
+# android studio config files
```

Comment:
I will need to expense IntelliJ Ultimate then.  Android Studio adds the file automatically.  It's also useful to work on the android modules and java module at the same time in Android Studio.